vault/token_store.go

```diff
@@ -2024,6 +2115,15 @@ func (ts *TokenStore) tokenStoreRoleCreateUpdate(
 }
 
 const (
+	tokenCleanupHelp = `
```

Comment:
I'd say "This endpoint performs cleanup tasks that can be run if certain error conditions have occurred."